README.md

```diff
@@ -86,17 +86,17 @@ httpServer.listen(3000);
 
 ```javascript
 angular.module('rethinkDBWorkshop.services', [])
-  .factory('messageFactory', refactoryFactory, function (refactoryFactory) {
+  .factory('messageFactory', ['refactoryFactory', function (refactoryFactory) {
       return refactoryFactory({
           model: 'message'
       });
-  })
-  .factory('userFactory', refactoryFactory, function (refactoryFactory) {
+  }])
+  .factory('userFactory', ['refactoryFactory', function (refactoryFactory) {
       return refactoryFactory({
           model: 'user'
       });
-  })
-  .controller('MainController', function (messageFactory) {
+  }])
+  .controller('MainController', ['messageFactory', function (messageFactory) {
 
     // Get the initial state of messages (might be an empty array)
     var messages = messageFactory();
```

Comment:
Hmm.  So any refactory factory is a function with added properties.  How does this use case differ from messageFactory.get()?  Is the point to initialize something synchronously while the data loads?